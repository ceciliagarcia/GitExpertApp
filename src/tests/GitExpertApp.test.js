import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import { GitExpertApp} from '../GitExpertApp';

describe('Prueba de <GitExpertApp />', () => {

    test('debe de mostrarse correctamente', () => {
        
       const  wrapper = shallow( <GitExpertApp /> );
       expect(wrapper).toMatchSnapshot();
    })

    test('debde de mostrar una lista de categorias', () => {
        const categories = ['Harry Potter', 'Sonic'];
        const  wrapper = shallow( <GitExpertApp defaultCategories={categories} /> );
        expect(wrapper.find('GifGrid').length).toBe(categories.length);
    })
    
    
})