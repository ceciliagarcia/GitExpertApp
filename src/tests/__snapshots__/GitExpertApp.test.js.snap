// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba de <GitExpertApp /> debe de mostrarse correctamente 1`] = `
<Fragment>
  <h2>
    GitExpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol />
</Fragment>
`;
